openmetadata-service/src/main/java/org/openmetadata/service/events/scheduled/PipelineServiceStatusJob.java

```diff
@@ -39,11 +46,35 @@ public void execute(JobExecutionContext jobExecutionContext) {
     }
   }
 
+  private String getServiceStatus(PipelineServiceClient pipelineServiceClient) {
+    RetryConfig retryConfig =
+        RetryConfig.<String>custom()
+            .maxAttempts(MAX_ATTEMPTS)
+            .waitDuration(Duration.ofMillis(BACKOFF_TIME_SECONDS * 1_000L))
+            .retryOnResult(response -> !HEALTHY_STATUS.equals(response))
+            .failAfterMaxAttempts(false)
```

Comment:
We check 3 times if the status is not unhealthy. If it still is, we return it to be registered in prometheus